File: biju-ngapp/biju-api/src/main/java/bijus/entity/Categoria.java
package bijus.entity;

public enum Categoria {

	BIJUTERIA("Bijuteria"),
	SEMIJOIA("Semij\u00F3ia"),
	JOIA("J\u00F3ia");
	
	
	private String descricao;
	
	
	Categoria(String descricao) {
		this.descricao = descricao;
	}
	
	public String getDescricao() {
		return descricao;
	}
	
//	public static void main(String[] args) {
//		List<Categoria>list = Arrays.asList(Categoria.values());
//	}
	
}

File: biju-ngapp/ngapp/src/vo/categoria.ts

export class Categoria {

    public readonly BIJUTERIA: String = "Bijuteria";
	public readonly SEMIJOIA: String = "Semij\u00F3ia";
	public readonly JOIA: String = "J\u00F3ia";

	public categoria: String
    
	public getCategorias(): Categoria[] {
		let categorias: Categoria[];

		let biju: Categoria = new Categoria();
		let semi: Categoria = new Categoria();
		let joia: Categoria = new Categoria();
		biju.categoria = this.BIJUTERIA;
		semi.categoria = this.SEMIJOIA;
		joia.categoria = this.JOIA;
	
		categorias.push(biju);
		categorias.push(semi);
		categorias.push(joia);

		return categorias;
	}
 
}